components/salesforce/sources/new-object/new-object.js

```diff
@@ -2,26 +2,31 @@ const startCase = require("lodash/startCase");
 
 const common = require("../../common");
 
-const EVENT_SOURCE_NAME = "New Object (Instant)";
-
 module.exports = {
   ...common,
-  name: EVENT_SOURCE_NAME,
+  name: "New Object",
```

Comment:
Is this the name Zapier assigns? Maybe we can edit the description to make the fact that you choose the object type clear

Reply:
They actually use `New Record in Salesforce`. We could rename it to that?